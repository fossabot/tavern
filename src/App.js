import React from 'nervjs'
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
import ActionHome from 'material-ui/svg-icons/action/home'
import c from 'classnames'

import s from './App.scss'

const App = () => (
  <MuiThemeProvider>
    <div className={c(s.App)}>
      <header className={c(s.header)}>
        <ActionHome />
        <h1 className={c(s.title)}>Welcome to React</h1>
      </header>
      <p className={c(s.intro)}>
        To get started, edit <code>src/App.js</code> and save to reload.
      </p>
    </div>
  </MuiThemeProvider>
)

export default App
