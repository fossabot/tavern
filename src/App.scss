.App {
  text-align: center;
}

.logo {
  animation: logo-spin infinite 20s linear;
  color: white;
  height: 80px;
}

.header {
  background-color: #222;
  padding: 20px;
  color: white;
}

.title {
  font-size: 1.5em;
}

.intro {
  font-size: large;
}

@keyframes logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}
