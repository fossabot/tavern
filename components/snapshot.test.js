/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'

import App from '../pages/index.js'

describe('With Snapshot Testing', () => {
  it('App match snapshot', () => {
    const component = renderer.create(<App />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
