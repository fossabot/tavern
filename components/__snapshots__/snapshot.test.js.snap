// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<div
  id="__layoutRoot"
>
  <header
    className="MuiPaper-root-9 MuiPaper-elevation4-15 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-7"
  >
    <div
      className="MuiToolbar-root-36"
    >
      <button
        aria-label="Menu"
        className="MuiButtonBase-root-44 MuiIconButton-root-38"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-43"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-47"
            color={undefined}
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-53"
        />
      </button>
      <p
        className="MuiTypography-root-60 MuiTypography-body1-69 MuiTypography-colorInherit-79"
        type="title"
      >
        <span
          id="toolbarTitle"
        >
          Home
        </span>
      </p>
    </div>
  </header>
  <h1
    className=""
  >
    Home
  </h1>
</div>
`;
